test/Core.Test/Services/GroupServiceTests.cs

```diff
@@ -30,12 +33,116 @@ public GroupServiceTests()
             );
         }
 
-        // Remove this test when we add actual tests. It only proves that
-        // we've properly constructed the system under test.
         [Fact]
-        public void ServiceExists()
+        public async Task SaveAsync_DefaultGroupId_CreatesGroupInRepository()
         {
-            Assert.NotNull(_sut);
+            var organization = new Models.Table.Organization
```

Comment:
It's OK to `using` these namespaces so you don't need a whole big namespace train.

If there's ambiguity, just resolve in place